pkg/services/authz/zanzana/common/info.go

```diff
@@ -20,6 +20,11 @@ var typedResources = map[string]typeInfo{
 		folderalpha1.FolderResourceInfo.GroupResource().Resource,
 		"",
 	): {Type: "folder", Relations: RelationsFolder},
+	FormatGroupResource(
+		"team.grafana.app",
+		"teams",
```

Comment:
These will live under iam app. We already have types and apis for them https://github.com/grafana/grafana/blob/main/pkg/apis/iam/v0alpha1/types_team.go.

So similar to folder we can import the api and get group and resource from there